lease/lessor.go

```diff
@@ -909,3 +909,9 @@ func (fl *FakeLessor) ExpiredLeasesC() <-chan []*Lease { return nil }
 func (fl *FakeLessor) Recover(b backend.Backend, rd RangeDeleter) {}
 
 func (fl *FakeLessor) Stop() {}
+
+type FakeTxnDelete struct {
+	backend.BatchTx
```

Comment:
Do we need to include backend BatchTx? Can we simply makes End() to do nothing?

Reply:
We need backend's lock.
- Benchmark save data to tmp file by backend, and backend only support unsafe op (no lock).
- Before save/delete data, we need call `RangeDeleter` to lock.
- After save/delete data, we need call `End` to unlock.

And use lock of backend  make bench more realistic.